oak_functions/loader/tests/integration_test.rs

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use lazy_static::lazy_static;
+use maplit::hashmap;
+use oak_functions_abi::proto::{Request, Response};
+use oak_functions_loader::{
+    logger::Logger,
+    lookup::{LookupData, LookupDataSource},
+    server::WasmHandler,
+};
+use std::{io::Write, sync::Arc};
+use tokio;
+
+lazy_static! {
```

Comment:
I think https://docs.rs/once_cell/latest/once_cell/ is the recommended way of doing static initialisation, these days (see https://news.ycombinator.com/item?id=24672238)

Reply:
done, thank you!